README.md

```diff
@@ -39,13 +38,13 @@ For Scala 2.11.x and 2.12.x:
 [comment]: # (Start Replace)
 
 ```scala
-libraryDependencies += "com.47deg" %% "iota-core" % "0.0.2"
+libraryDependencies += "com.47deg" %% "iota-core" % "0.1.0"
```

Comment:
BTW, the README file in the root is also automatically updated by `tut` and commited by sbt-org-policies when the one in the tut sources is compiled.